frontend/src/views/Combat.vue

```diff
@@ -69,6 +69,13 @@
                 <b-button v-if="selectedWeaponId" variant="primary" class="ml-3" @click="selectedWeaponId = null" id="gtag-link-others" tagname="choose_weapon">
                   Choose New Weapon
                 </b-button>
+
+                <b-button v-if="selectedWeaponId" variant="primary" class="ml-3 smoke-bomb-button"
+                @click="useSmokeBombUI()" id="gtag-link-others" tagname="use_smokebomb"
```

Comment:
@click="useSmokeBombUI" <- you can delete () here

Reply:
done